tasks/global-setup-windows.yml

```diff
@@ -23,6 +23,19 @@
     - restart_gitlab_runner_macos
     - restart_gitlab_runner_windows
 
+- name: (Windows) Set check_interval option
+  lineinfile:
+    dest: "{{ gitlab_runner_config_file }}"
+    regexp: ^(\s*)check_interval =
+    line: \1check_interval = {{ gitlab_runner_check_interval }}
```

Comment:
What is `\1` doing?

Reply:
It's a backreference to the whitespace matching group `(\s*)`. I believe it's there so that the new line retains the same whitespace indent as before (disclaimer: I copied this from the other tasks before and after for consistency).

Reply:
Looked at it again and it's really just overcomplicating things, so have removed this. I couldn't see a compelling reason why this is needed (except maybe for the first config value in this file?).